Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Params
+ARG RAILS_ENV="production"
+
+# Defining ruby version
+FROM ruby:2.6.6
```

Comment:
I would parametarise the ruby version so that we have control over it. Try this:
```
ARG RUBY_VERSION=2.6.6 
FROM ruby:$RUBY_VERSION
```